tests/sles4sap/netweaver_cluster.pm

```diff
@@ -24,7 +24,7 @@ sub run {
     my $type         = get_required_var('INSTANCE_TYPE');
     my $sid          = get_required_var('INSTANCE_SID');
     my $alias        = get_required_var('INSTANCE_ALIAS');
-    my $lun          = get_required_var('INSTANCE_LUN');
+    my $lun          = get_var('INSTANCE_LUN');
```

Comment:
Why removing the `required` part? As this LUN is needed in the cluster.

Reply:
Not on HANA. I could change it to required on NW.

Reply:
Ok, as far as I remember this variable is set between 2 test files, not in the test suite. So in that case the `get_var` could be sufficient. Will see with the regression tests.